### ISSUE TITLE
Training Problems

### ISSUE THREAD

**Issue by yinyiyu** [Ha0Tang/XingGAN#4]
@Ha0Tang Thanks for your novel work. I have trained the marker dataset follow your guidance. But I have a question that With the increase of training iteration, the loss of each part also increases except D_PP and D_PB。what are the two parts mean? I also wanted to ask how many epochs were used in the pre-training models you provided
![image](https://user-images.githubusercontent.com/24800963/94422310-7e089000-01b9-11eb-89cc-7dcb9f8c3cd3.png)
![image](https://user-images.githubusercontent.com/24800963/94422359-8b257f00-01b9-11eb-8b2e-6ae741bc8380.png)

**Comment by Ha0Tang** [comment]
1. D_PP is the appearance discriminator as mentioned in the paper, D_PB is the shape discriminator.
Please refer to [these lines](https://github.com/Ha0Tang/XingGAN/blob/3df7a9839e179cbf53fcb599e3287dd1f46d4e10/models/XingGAN.py#L182) for more details. 

2. In the ReadMe, we have provided the training and test scripts which contain the hyper-parameters we adopted in our experiments and you should achieve similar results as our provided model.
